<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>لیست روز ها</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/icon.ico</normaloff>:/icon/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4" rowstretch="25,75">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="PbUpdate">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>60</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>16</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>بروزرسانی</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="50,20,30">
           <item>
            <widget class="QLabel" name="lebTimeTotal">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>4</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lebTime">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>1</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>زمان کل</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="50,20,30">
           <item>
            <widget class="QLabel" name="lebGhoflTotal">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>5</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lebGhofl">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>2</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>مجموع قفل</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="3" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="50,50">
           <item>
            <widget class="QLabel" name="allTotal">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>7</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>جمع کل</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="50,20,30">
           <item>
            <widget class="QLabel" name="lebSnappTotal">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>6</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lebSnapp">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>3</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>رفت و آمد</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>300</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout" columnstretch="30,70">
      <property name="horizontalSpacing">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <widget class="QTableView" name="tableView_2">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="showGrid">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>false</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>130</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableView" name="tableView">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>80</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>50</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Day"/>
    <addaction name="actionAdd_Snapp"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_items"/>
    <addaction name="actionDelete_items"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_numbers"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSize_Window"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_Day">
   <property name="text">
    <string>Add Day</string>
   </property>
  </action>
  <action name="actionAdd_Snapp">
   <property name="text">
    <string>Add Snapp</string>
   </property>
  </action>
  <action name="actionEdit_numbers">
   <property name="text">
    <string>Edit numbers</string>
   </property>
  </action>
  <action name="actionSize_Window">
   <property name="text">
    <string>Size Windows</string>
   </property>
  </action>
  <action name="actionDelete_items">
   <property name="text">
    <string>Delete items</string>
   </property>
  </action>
  <action name="actionEdit_items">
   <property name="text">
    <string>Edit items</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
